libs/auth/src/lib/components/register/register.component.html

```diff
@@ -14,48 +14,92 @@ <h1 class="w-full my-4 text-lg font-bold text-center">Mebli</h1>
                  name="email"
                  type="email"
                  [placeholder]="'email_placeholder' | transloco"
+                 pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$" 
```

Comment:
Hier bekomme ich Fehler in der Entwickler Konsole. Bitte mal überprüfen:
ERROR SyntaxError: Invalid regular expression: /^((?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,30}+\W)$/: Nothing to repeat

Reply:
Error behoben